AirCasting/CreateSessionViews/SelectDeviceView.swift

```diff
@@ -84,6 +87,10 @@ struct SelectDeviceView: View {
             createLabel(title: Strings.SelectDeviceView.bluetoothLabel, subtitle: Strings.SelectDeviceView.bluetoothDevice)
         })
         .buttonStyle(WhiteSelectingButtonStyle(isSelected: selected == 1))
+        .disabled(!isBTButtonAvailable)
+        .onTapGesture {
+            alert = InAppAlerts.bluetoothSessionAlreadyRecordingAlert()
```

Comment:
Will it do what I think it will do? When user taps a disabled button it will show an alert? Is that good UX? I'm not sure... I think very few users would actually tap on a disabled button. If we want to explain why the button is disabled we should add a label.

disregard if this is not the case here (IDK how `WhiteSelectingButtonStyle` looks like TBH)

Reply:
Thanks! It will be changed